Maple2.Model/Metadata/Table/FishTable.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections.Generic;
+
+namespace Maple2.Model.Metadata;
+
+public record FishTable(IReadOnlyDictionary<int, FishTable.Entry> Entries) : Table {
+    public record Entry(int Id,
+                        string FluidHabitat,
+                        IReadOnlyList<int> HabitatMapIds,
+                        int Mastery,
+                        short Rarity,
+                        int[] SmallSize,
```

Comment:
These arrays are always 2 values right? If that's true it might be good to define a new type:
```cs
...
FishTable.Range SmallSize,
FishTable.Range BigSize);

public record struct Range(int Min, int Max);
```

ref: https://github.com/kOchirasu/Maple2/blob/master/Maple2.Model/Metadata/Table/ItemOptionTable.cs#L35